app/src/main/java/com/hedvig/app/feature/insurance/ui/InsuranceViewModel.kt

```diff
@@ -0,0 +1,49 @@
+package com.hedvig.app.feature.insurance.ui
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
+import com.hedvig.android.owldroid.graphql.InsuranceQuery
+import com.hedvig.app.feature.insurance.data.InsuranceRepository
+import e
+import kotlinx.coroutines.launch
+
+abstract class InsuranceViewModel : ViewModel() {
+    abstract val data: MutableLiveData<Result<InsuranceQuery.Data>>
```

Comment:
A really cool thing we could do is this:

```suggestion
    protected val _data = MutableLiveData<Result<InsuranceQuery.Data>>
    val data: LiveData<Result<InsuranceQuery.Data>> = _data
```